src/evaluation/errors.rs

```diff
@@ -16,6 +16,6 @@ pub enum EvaluationError {
     #[error("WebAssembly failure: {0}")]
     WebAssemblyError(String),
 
-    #[error("{0}")]
-    InternalError(String),
+    #[error("Group policy used as individual policy: {0}")]
+    GroupPolicyUsedAsIndividualPolicy(String),
```

Comment:
This error is only returned if someone tries to rehydrate a policy group.
Could we be more explicit?

```rust
    #[error("Cannot rehydrate policy group: {0}")]
    CannotRehydratePolicyGroup(String),
```
Where the `String` is the policy_id and not a message